projects/core/src/auth/user-auth/config/auth-config.ts

```diff
@@ -90,6 +90,11 @@ export abstract class AuthConfig {
      * Config for angular-oauth-oidc library.
      */
     OAuthLibConfig?: AuthLibConfig;
+
+    customLoginPage?: {
+      csrfEndpoint?: string;
+      loginFormEndpoint?: string;
+    };
```

Comment:
`medium:` Let's add description for these config params, for customers and for us from the future